src/assets/Generator.Shared/OperationFinalStateVia.cs

```diff
@@ -7,8 +7,10 @@ namespace Azure.Core
 {
     internal enum OperationFinalStateVia
     {
+        LocationOverride,
         AzureAsyncOperation,
         Location,
-        OriginalUri
+        OriginalUri,
+        OperationLocation
```

Comment:
Tip: I recommend adding a terminal comma so that future edits are only new lines. Makes for a cleaner diff.